launch_pilot.py

```
@@ -24,10 +29,406 @@
+        "wind_speed": wind_speed
+    }
+
+def check_thermal_alerts(thermal_data: Dict[str, float]) -> List[Dict[str, str]]:
+    """
+    Generate alert records for thermal measurements that violate configured thresholds.
+    
+    Parameters:
+        thermal_data (Dict[str, float]): Thermal measurements with required keys:
+            - "panel_temp": panel temperature in degrees Celsius.
+            - "inverter_temp": inverter temperature in degrees Celsius.
+            - "efficiency": panel efficiency as a fraction (0.0–1.0).
```

Ruff (0.13.3)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)